gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -478,6 +492,8 @@ def run(argv=None):
                num_bigquery_write_shards=known_args.num_bigquery_write_shards,
                null_numeric_value_replacement=(
                    known_args.null_numeric_value_replacement)))
+      if known_args.generate_call_info_table:
+        _create_call_info_table(pipeline, pipeline_mode, known_args)
```

Comment:
Is this the best place to do this? How about somewhere like `_merge_headers` which we are reading headers anyways to avoid reading headers multiple times?

Reply:
I thought about reusing the code in `_merge_headers`. The thing is, if `--representative_header_file` is supplied, we won't merge headers. I feel it won't be clean if this step is also added there (a bunch of if-else cases have to be added). Also, merge_headers is a separate pipeline (different dataflow job), it generates the `representative_header_file` for the main pipeline (vcf to bq). If we also write the meta info to BQ in merge_headers, the logic is not clean. Even worse, imaging some failures happen in the main pipeline, it would be difficult to roll back the changes for this table to keep appending for all tables as an atomic operation. Does this make sense to you?